docs/ODESwagger.yaml

```diff
@@ -107,30 +107,177 @@ paths:
         400:
           description: Value input error
 definitions:
+  TravelerMessage:
+    type: object
+    properties:
+      timeContent:
+         $ref: '#/definitions/TimContent'
+      RSUs:
+        type: array
+        items:
+           $ref: '#/definitions/RSU'
+      snmp: 
+         $ref: '#/definitions/SNMP'
+  TimContent:
+    type: object
+    required: [msgCnt]
+    properties:
+      msgCnt:
+        type: integer 
+      timestamp:
+        type: integer
+      packetId: 
+        type: integer
+      urlB: 
+        type: string
+        description: UrlB
+      travelerDataFrames: 
+        type: array
+        items: 
+          $ref: '#/definitions/TravelerDataFrame'
   TravelerDataFrame:
     type: object
     properties:
-      sspTimRights:
+      header:
+        $ref: '#/definitions/TravelerDataFrameHeader'
+      region:
+        $ref: '#/definitions/TravelerDataFrameRegion'
+      content:
+        $ref: '#/definitions/TravelerDataFrameContent'
+  TravelerDataFrameHeader:
+    type: object
+    required : [sspindex, travelerInfoType, SignPriority, MinuteOfTheYear]
+    properties:
+      sspindex:
         type: string
         description: The SSP index is used to control the data elements that follow the occurrence of the index. The index relates back to the SSP contents in the CERT used to declare what content is allowed by that CERT. In the absence of a matching index in the message sender’s CERT, the message contents are not valid.
-      frameType:
+      travelerInfoType:
         type: string
         description: The DE_TravelerInfoType data element provides the type of message to follow in the rest of the message frame structure. It is used in the traveler information message, which may contain several such structures.
       msgId:
-        type: string
-        description: Link to other messages
-      startYear:
-        type: string
+        $ref: '#/definitions/MsgID'
+      DYear:
+        type: integer
         description: The DSRC year consists of integer values from zero to 4095 representing the year according to the Gregorian calendar date system. The value of zero shall represent an unknown value.
-      startTime:
-        type: string
+      MinuteOfTheYear:
```

Comment:
Inconsistent naming. All property names start with lowercase, similar to java property naming convention. This and the next violate that convention. Also, I do not recommend changing the property name from ASN.1 names if we are not changing the semantics of it. If the value is still going to be minutes of the year, I would preserve the startTime as the property name. Same goes with durationTime.